src/epiviz.gl/utilities.js

```diff
@@ -311,17 +311,33 @@ const getPointsBySelectMode = (selectMode, originalPoints, xRange, yRange) => {
   return points;
 };
 
+const throttleWithRAF = (callback) => {
```

Comment:
needs documentation for this function, also what does RAF mean?

Reply:
Updated & added documentation, RAF = request animation frame.